CHANGELOG.rst

```diff
@@ -1,6 +1,19 @@
 Changelog
 =========
 
+0.8.2 (2019-01-28)
+------------------
+
+MLflow 0.8.2 is a patch release on top of 0.8.1 containing only bug fixes and no breaking changes or features.
+
+Bug fixes:
+
+- [Python API] CloudPickle has been added to the set of MLflow library dependencies, fixing missing import errors when attempting to save models (#777, @tmielika)
```

Comment:
MLflow does not specify a version requirement for the CloudPickle dependency; users will only encounter issues if they are running the latest version that is not yet available on Anaconda. Rather than pinning CloudPickle, I think it makes sense to push them to upload the latest version to Anaconda.